### PR TITLE
Version one

### PR DESCRIPTION
**Pull request by RoyNyaga**
There are some comments i left in my html and css files, it's for the purpose of easy review. Will clean that up when the project is accepted.

### CODE REVIEW

**Comment by Ispirett** [comment]
Hey, @RoyNyaga  great work with this project you did a  fantastic job :1st_place_medal: Also love your read me file :+1: .  Just some minor fixes 

### HTML errors
There are a  few  HTML errors on this page use this tool and please correct them https://validator.w3.org/.

**When finished you can submit for review.**

### Optional changes
Using `<section>` tags instead of `<div>` for groups of related articles is good practice.
`<.list-links >` for margin-right consider using shorthand syntax like `magin: 10px 10px 10px 10px `            and padding-right is overidering another decaration high up in the that class consider removing one.


You can consider adding this section to give your site to get that complete pristine look :100:  
bootstrap

**Comment by RoyNyaga** [comment]
I rectified all the errors thrown by the markup validator and implemented section tags for main sections in the main tag. I also added the last section of the main tag as requested. I purposely left comments in my work for easy review in case there are any.

**Comment by bolah2009** [comment]
Currently under review

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+}
+
+.header-nav {
+  padding: 15px 160px 15px 0;
+  border-bottom: 2px solid #d9d9d9;
+  height: 56px;
+  background-color: white;
+}
+.header-nav::before {
+  position: absolute;
```
Comment (bolah2009) [inline css/style.css:65]: `SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+  left: 0;
+  z-index: 1;
+}
+.category a {
+  color: #f72210;
+  font-size: 14px;
+}
+
+.header-article {
+  position: relative;
```
Comment (bolah2009) [inline css/style.css:136]: `SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+.news-btn {
+  padding: 10px 10px 10px 12px;
+  color: #fff;
+  margin-bottom: 15px;
+  background-color: #666;
+  font-size: 18px;
+}
+.news-btn::after {
+  content: "";
+  display: inline-block;
```
Comment (bolah2009) [inline css/style.css:272]: `SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome Job [Roy](../../) :clap: 

Awesome job so far, the [clone](https://raw.githack.com/RoyNyaga/using_bootstrap/version_one/index.html) looks good to me and very responsive.. :clap:

However, use more of bootstrap as pointed out in [this comment](https://github.com/RoyNyaga/using_bootstrap/pull/1#discussion_r311328999) and all other occurrences and make the navbar fixed as in the [original site](https://www.newsweek.com/), this is to make the homepage and website menu accessibility anyway in the page especially for long websites which is the case in this website. See below for illustration:

|Current|Preffered|
| :-----: | :------: |
| newsweek fix navbar cureent | newsweek fix navbar preffred |

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Comment by RoyNyaga** [comment]
Added the bootstrap navbar-fixed to create a fixed nav and added more bootstrap classes as demanded. Notice in some areas i couldn't use the bootstrap margin and padding because the highest it could produce (ie m-5) was not enough to produce the effect i wanted. Bootstrap colors are a little bit limited to their default colors, so i had to use CSS colors in some areas.

**Comment by samgaco** [comment]
### Hi @RoyNyaga :hand: 

### Good work implementing the changes required in the previous reviews. Congratulations because your project is approved :+1: :tada: 

**[OPTIONAL]** _There are some things that you could still improve like the icons at the footer (they should arrange below the logo in the smaller screen size, or the opinion section in the medium screen size get's too much empty space at its right. Those would be things to work on if you would like to keep digging into this project._

### You may merge to the master branch and submit the project completion form. Keep the good work for the next projects :muscle: Regards!.

**Comment by RoyNyaga** [comment]
Ok thanks for the quick review, i will try and improve the icons at the
footer for smaller screens before submitting.

**Pull request merged**